tests/hpc/slurm_master_db.pm

```diff
@@ -18,6 +18,165 @@ use warnings;
 use testapi;
 use lockapi;
 use utils;
+use Data::Dumper;
+use Mojo::JSON;
+
+## TODO: provide better parser for HPC specific tests
+sub get_test_name {
+    return my $name = (caller(2))[3];
```

Comment:
```suggestion
    return (caller(2))[3];
```

should work too , right ?

Reply:
Not sure if it is a good idea to rely on the call stack, to get a specific name. I would go with specifing it or use a global variable to account like `'test_' . $X++;`

Reply:
yeah, this is obviously not nice and frankly it was my learning a bit about perl ;-)

I switched to $name as this is much cleaner and goes well with Anton's comment which I agree with.